app/javascript/components/DragAndDropOnField.jsx

```diff
@@ -42,18 +31,10 @@ export default class DragAndDropOnField extends Component {
             this.props.enemyPlayers
           );
           this.props.onHandleUpdateOldPlayersPosition(oldPosition);
-          this.props.onHandleDraging(false);
         }}
       >
-        <EnemyTshirt/>
-        <Text
-          x={player.id < 10 ? 23 : 17}
-          y={12}
-          text={player.id}
-          fontSize={20}
-          fill="white"
-          fontStyle="bold"
-        />
+        <Text text="👚" fontSize={50} />
```

Comment:
did you resolved conflicts correctly in this file? I believe that team decided at some point to not to use emoji as t-shirt images. 

@kcc112 @olgapps any opinion on that?